### PR TITLE
ref(replays): update widgets so that only nonzero rows are displayed

### PR DESCRIPTION
**Pull request by michellewzhang**
closes https://github.com/getsentry/team-replay/issues/192 
SCR-20230925-jooq
SCR-20230925-jorj

### CODE REVIEW

File: static/app/components/gridEditable/index.tsx [hunk 1]
```diff
@@ -415,7 +415,7 @@ class GridEditable<
     return (
       <GridRow>
         <GridBodyCellStatus>
-          <EmptyStateWarning>
+          <EmptyStateWarning small>
```
Comment (michellewzhang) [inline static/app/components/gridEditable/index.tsx:418]: default size was too large to fit in the same box size as our widget

File: static/app/components/gridEditable/styles.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import {space} from 'sentry/styles/space';
 
 export const GRID_HEAD_ROW_HEIGHT = 45;
 export const GRID_BODY_ROW_HEIGHT = 40;
-export const GRID_STATUS_MESSAGE_HEIGHT = GRID_BODY_ROW_HEIGHT * 4;
+export const GRID_STATUS_MESSAGE_HEIGHT = GRID_BODY_ROW_HEIGHT * 3;
```
Comment (michellewzhang) [inline static/app/components/gridEditable/styles.tsx:9]: we could avoid this change by fetching 4 rows of selectors rather than 1

Comment (ryan953) [reply static/app/components/gridEditable/styles.tsx:9]: Does this impact other places around the app that use gridEditable?

File: static/app/components/gridEditable/index.tsx (hunk 1 above)
Comment (ryan953) [reply static/app/components/gridEditable/index.tsx:418]: Does this impact other places around the app that use gridEditable?

File: static/app/components/gridEditable/styles.tsx (hunk 2 above)
Comment (ryan953) [reply static/app/components/gridEditable/styles.tsx:9]: im ok with 4 rows :D

Comment (michellewzhang) [reply static/app/components/gridEditable/styles.tsx:9]: yes but they look the same as the screenshots above:

SCR-20230925-jnmz

SCR-20230925-jnuy

Comment (michellewzhang) [reply static/app/components/gridEditable/styles.tsx:9]: gotcha let me change to 4 rows

**Review by ryan953 — APPROVED** [review]

File: static/app/components/gridEditable/styles.tsx [hunk 3]
```diff
@@ -242,7 +242,6 @@ const GridStatusFloat = styled('div')`
   width: 100%;
   height: ${GRID_STATUS_MESSAGE_HEIGHT}px;
   overflow: hidden;
-
```
Comment (ryan953) [inline static/app/components/gridEditable/styles.tsx:245]: we can remove this change to keep git history clean

File: static/app/components/gridEditable/styles.tsx [hunk 4]
```diff
@@ -242,7 +242,6 @@ const GridStatusFloat = styled('div')`
   width: 100%;
   height: ${GRID_STATUS_MESSAGE_HEIGHT}px;
   overflow: hidden;
```
Comment (ryan953) [inline static/app/components/gridEditable/styles.tsx:244]:
```suggestion
  overflow: hidden;

```

File: static/app/components/gridEditable/styles.tsx (hunk 3 above)
Comment (michellewzhang) [reply static/app/components/gridEditable/styles.tsx:245]: 🫡

**Pull request merged**